### ISSUE TITLE
intermittent `error occured`

### ISSUE THREAD

**Issue by kosich** [observable-playground/observable-playground#20]
> An error occurred loading this page's template. More information is available in the console.

```
vendors~main.f60db702.js:1 Possible Unhandled Promise Rejection: Error: export not found
```

Possible solution: move sync execution to a webworker

**Comment by kosich** [comment]
attempted to fix this in `f9b5e1d4cc84efb68e460d4448dcb28d713ecb0c`. trying to reproduce with a fix.

**Comment by kosich** [comment]
It is still present

**Comment by kosich** [comment]
Probably fixed with #24
